clients/lsp-camel.el

```diff
@@ -20,9 +20,8 @@
 ;; Directory in which the servers will be installed. Lsp Server Install Dir: ~/.emacs.d/.cache/camells
 (defcustom lsp-camel-jar-file (f-join lsp-server-install-dir "camells" lsp-camel-jar-name)
   "Camel Language server jar command."
-  :type 'string
   :group 'lsp-camel
-  :type 'file
+  :type 'directory
```

Comment:
this should be `file` instead of `directory`, no?

Reply:
Ah, got confused from the comment above and seeing the word `dir` too many times. Definitely should be `file`